static/Makefile

```diff
@@ -40,12 +42,14 @@ clean: ## remove build artifacts
 static: static-linux cross-mac cross-win cross-arm ## create all static packages
 
 .PHONY: static-linux
-static-linux: static-cli static-engine ## create tgz with linux x86_64 client and server
+static-linux: static-cli static-engine static-buildx-plugin ## create tgz
 	mkdir -p build/linux/docker
 	cp $(CLI_DIR)/build/docker build/linux/docker/
 	for f in dockerd containerd ctr containerd-shim containerd-shim-runc-v2 docker-init docker-proxy runc; do \
 		cp -L $(ENGINE_DIR)/bundles/binary-daemon/$$f build/linux/docker/$$f; \
 	done
+	mkdir -p build/linux/docker/cli-plugins
+	cp $(BUILDX_DIR)/bin/docker-buildx build/linux/docker/cli-plugins/
```

Comment:
As discussed, not sure if we should include buildx in the docker-ce-cli static pkg even if it's a required component now.

Reply:
Removed for now